src/shared/alias-mappings.ts

```diff
@@ -1,8 +1,20 @@
 export enum AliasName {
 	YEAR1 = "year1",
+	Y1P1 = "y1p1",
+	Y1P2 = "y1p2",
+	Y1P3 = "y1p3",
+	Y1P4 = "y1p4",
```

Comment:
yikes.
I am gonna leave this for now and have a look some other day.

Reply:
haha